tests/flow/test_constraint.py

```diff
@@ -263,6 +263,31 @@ def test02_constraint_violations(self):
         actual_result_set = self.g.query("MATCH (n) RETURN n ORDER BY ID(n)").result_set
         self.env.assertEqual(actual_result_set, expected_result_set)
 
+        try:
+            g.query("CREATE CONSTRAINT ON (n:N) ASSERT n.v IS UNIQUE")
```

Comment:
Do you think we should support this variation ?
Similar to the `CREATE INDEX FOR...` queries ?

Reply:
no I prefer to remove this from the parser but this gives us option to return good error